frontend/test/reducers/reducers.spec.js

```diff
@@ -0,0 +1,13 @@
+import rootReducer from '../../reducers';
+import { REQUEST_CHANNEL_PLATFORM_SUMMARY_DATA } from '../../actions';
+
+describe('channelPlatformSummary reducer', () => {
+    it('Requests channel platform summary data', () => {
+        const initialState = { };
+        const action = { type:REQUEST_CHANNEL_PLATFORM_SUMMARY_DATA  };
+        const expectedState = { channelPlatformSummary: { isFetching: true },
+                             measures: {} }
```

Comment:
the linting here isn't quite right, you should run `yarn lint --fix` to correct it.